sros2/test/sros2/commands/security/verbs/test_amend_policy_cli.py

```diff
@@ -0,0 +1,66 @@
+# Copyright 2019 Canonical Ltd
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import builtins
+import mock
+import os
+import rclpy
+import tempfile
+
+from shutil import copyfile
+from test import get_policies_path
+
+from ros2cli import cli
+from sros2.policy import load_policy
+from std_msgs.msg import String
+
+
+def test_ament_policy():
+    with tempfile.TemporaryDirectory() as tmpdir:
```

Comment:
I know you're copying other tests here, but I've recently learned a better way to do this: [pytest's `tmp_path` fixture](https://docs.pytest.org/en/latest/tmpdir.html#the-tmp-path-fixture). Try this:

```python3
def test_amend_policy(tmp_path):
    # ...
    policy_generated_test_file = str(tmp_path.joinpath('dummy_policy.xml'))
    # ...
```